sampledb/frontend/utils.py

```diff
@@ -217,3 +217,35 @@ def filter_are_conditions_fulfilled(data, property_schema) -> bool:
 _jinja_filters['babel_format_number'] = custom_format_number
 _jinja_filters['base64encode'] = base64encode
 _jinja_filters['are_conditions_fulfilled'] = filter_are_conditions_fulfilled
+
+
+def get_template(template_folder, schema):
+    path = os.getcwd() + '/sampledb/frontend/templates/' + template_folder
```

Comment:
The current working directory might be a different path, so it would be better to use the global `__file__` variable that contains the path to the current Python module, e.g. something like:
`os.path.join(os.path.dirname(__file__), 'templates', template_folder)`